ui/app/styles/core/buttons.scss

```diff
@@ -3,12 +3,17 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// Button specific vars
 $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
 
+// ARG TODO is-focused is-hover are those classes added by a third party? I don't see them.
```

Comment:
Does anyone know the answer to this. We have classes called is-hover and is-focused but we never actually use them in the hbs. I'd like to remove them if this was just a class copy syntax we got from bulma (which is what I suspect happened).